os/include/tinyara/mpu.h

```diff
@@ -34,6 +34,16 @@
  * Pre-processor Definitions
  ********************************************************************************/
 
+struct mpu_regs {
+	uint8_t board_mpu_regs;
```

Comment:
What does the member mean? The number of mpu region for each? The highest number for each? The start number?

Reply:
this is the number of MPU reserved regions for the vendor